nwbinspector/checks/nwb_containers.py

```diff
@@ -19,11 +19,18 @@ def check_dataset_compression(nwb_container: NWBContainer, gb_severity_threshold
         if (
             isinstance(field, h5py.Dataset)
             and field.compression is None
-            and field.nbytes > (gb_severity_threshold * 1e9)
+            and field.size * field.dtype.itemsize > 50 * 1e6  # 50 MB lower bound
```

Comment:
Note that `field.nbytes` only works with rather recent versions of `h5py`. This package inherits it's `h5py` version support range from `pynwb`, which includes versions that do not support that. But they all support `size` * `itemsize` operation.

Reply:
Specifically, `pynwb` still supports `<v3.0`, feature was added in exactly `v3.0`.

Reply:
ok